recipes/pycalphad/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.4.1" %}
+{% set sha256 = "d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
+package:
+  name: pycalphad
+  version: {{ version }}
+
+source:
+  fn: pycalphad-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pycalphad/pycalphad-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - matplotlib
+    - pandas
+    - nose
+    - mock
+    - xarray !=0.8
```

Comment:
pydata/xarray#943

On Aug 8, 2016 10:31 PM, jakirkham <notifications@github.com> wrote:

In recipes/pycalphad/meta.yaml<https://github.com/conda-forge/staged-recipes/pull/1195#discussion_r73989848>:

> +  url: https://pypi.io/packages/source/p/pycalphad/pycalphad-{{ version }}.tar.gz
> +  sha256: {{ sha256 }}
> +
> +build:
> +  number: 0
> +  script: python setup.py install --single-version-externally-managed --record record.txt
> +
> +requirements:
> +  build:
> +    - python
> +    - setuptools
> +    - matplotlib
> +    - pandas
> +    - nose
> +    - mock
> +    - xarray !=0.8


What is the reason for not using this particular version?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/conda-forge/staged-recipes/pull/1195/files/32a32a26c188e83b60593bde36c17b70c56fcd07#r73989848>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AGG9hlbmmXCF7XX2IMH1d01hK53uknQcks5qd-Z_gaJpZM4JfjnW>.